templates/partial/_careers-send-cv.html

```diff
@@ -3,19 +3,19 @@
     <div class="col-12">
       <h2>Can't find the ideal role for you?</h2>
       <p>Don't worry if you can't find the a role that fits you, send us your CV and we'll keep it on file to match any future roles to your skills to see if we can get a great match.</p>
-      <form>
+      <form action="{{request.path}}" method="POST" enctype="multipart/form-data">
```

Comment:
I would change `{{ request.path }}` to `/careers/thank-you` - see https://github.com/canonical-web-and-design/canonical.com/pull/38/files#r323151734